deploy/cephcsi/image/Dockerfile

```diff
@@ -38,6 +38,7 @@ RUN ${GOROOT}/bin/go version && ${GOROOT}/bin/go env
 
 RUN dnf -y install --nodocs \
 	librados-devel librbd-devel \
+    librados-devel libcephfs-devel \
```

Comment:
`librados-devel` is now listed 2x. Just add `libcephfs-devel` on the same line as `librbd-devel`